### PR TITLE
Add compat bounds for JLL packages compiled with SCOTCH_jll

### PR DESCRIPTION
**Pull request by amontoison**
@giordano

### CODE REVIEW

File: jll/M/MMG_jll/Compat.toml [hunk 1]
```diff
@@ -1,3 +1,4 @@
 [5]
 JLLWrappers = "1.2.0-1"
 julia = "1.6.0-1"
+SCOTCH_jll = "6.1.3-1"
```
Comment (giordano) [inline jll/M/MMG_jll/Compat.toml:4]: -1??

Comment (amontoison) [inline jll/M/MMG_jll/Compat.toml:4]:
```suggestion
SCOTCH_jll = "6.1.3-6"
```

File: jll/M/MUMPS_jll/Compat.toml [hunk 2]
```diff
@@ -9,6 +9,7 @@ JLLWrappers = "1.2.0-1"
 
 ["5.4.2-5"]
 julia = "1.6.0-1"
+SCOTCH_jll = "6.1.3-1"
```
Comment (amontoison) [inline jll/M/MUMPS_jll/Compat.toml:12]:
```suggestion
SCOTCH_jll = "6.1.3-6"
```

File: jll/N/neper_jll/Compat.toml [hunk 3]
```diff
@@ -2,3 +2,4 @@
 GSL_jll = "2.7.2-2.7"
 JLLWrappers = "1.2.0-1"
 julia = "1.6.0-1"
+SCOTCH_jll = "6.1.3-1"
```
Comment (amontoison) [inline jll/N/neper_jll/Compat.toml:5]:
```suggestion
SCOTCH_jll = "6.1.3-6"
```

File: jll/P/PTSCOTCH_jll/Compat.toml [hunk 4]
```diff
@@ -1,5 +1,6 @@
 [6]
 julia = "1.6.0-1"
+SCOTCH_jll = "6.1.3-1"
```
Comment (amontoison) [inline jll/P/PTSCOTCH_jll/Compat.toml:3]:
```suggestion
SCOTCH_jll = "6.1.3-6"
```

File: jll/Q/qr_mumps_jll/Compat.toml [hunk 5]
```diff
@@ -6,3 +6,6 @@ julia = "1"
 
 ["3.0.4-3"]
 julia = "1.6.0-1"
+
+["3.0.2-3"]
+SCOTCH_jll = "6.1.3-1"
```
Comment (amontoison) [inline jll/Q/qr_mumps_jll/Compat.toml:11]:
```suggestion
SCOTCH_jll = "6.1.3-6"
```

**Pull request merged**